.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+> __ALL Pull-Requests require an associated [Issue](https://github.com/thegooddocsproject/templates/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) as documented in our [Contributing Guide](https://github.com/thegooddocsproject/templates/blob/master/contribute.md#contributing).__
+
+## Purpose / Why
+
+> _**Replace This Text** - Restate the purpose/jusitifcation of this work and include links to the Issue inside of the purpose/why._
+
+## What Changes Were Made?
+
+> _**Replace This Text** - State clearly the direct additions or modifications made in this change-request._
+
+## Verification
+
+> _**Replace This Text** - Document the details that help a reviewer understand what steps or actions should be taken in-order to verify the work is completed and satisfies the initial scope of work._
+
+---
+
+## Type of Change
```

Comment:
Debating if __Type of Change__ should sit higher up in the Template, it is nice to know what exactly this change-type is before really digging into a PR.

Reply:
I'd err toward suggesting we don't include Type of Change, at least not in our first iteration. I'm not sure how close these categories will align with future work we do. And I wonder how much value it adds, verses the confusion it introduces.

Reply:
Seconding this. It seems to me that categorizing is what labels are for and then it's more of a triage task for reviewers. Might be better suited as a checklist item, below (e.g., "Is this PR labeled correctly?")

Reply:
We can hold off and I'll remove the section.  Labels will work for now.